DESIGN.md

```diff
@@ -1,3 +1,11 @@
+# Overview
+
+SAM is invoke/called by the CloudFormation Service. CloudFormation recognises the `Transform: AWS::Serverless-2016-10-31` header and invokes the SAM translator. This will then take your SAM template and expand it
```

Comment:
invoke -> invoked? Or maybe just drop and do “SAM is called by...”

Reply:
Updated in next commit.